packages/m2m-gateway/src/api/attributeApiConverter.ts

```diff
@@ -47,3 +47,38 @@ export function toM2MGatewayApiCertifiedAttribute({
     }
   }
 }
+
+export function toM2MGatewayApiDeclaredAttribute({
```

Comment:
Once https://github.com/pagopa/interop-be-monorepo/pull/2091 is merged you can reuse the converter that @crasherbit already added there ⬇️
https://github.com/pagopa/interop-be-monorepo/pull/2091/files#diff-b715216be9c35bca5cf0b2004ca696a4cec88d8ce2e17eca0dc87b2238005014R98